src/Microsoft.DotNet.ProjectModel/AnalyzerOptions.cs

```diff
@@ -15,7 +16,7 @@ public class AnalyzerOptions
 
         public static bool operator ==(AnalyzerOptions left, AnalyzerOptions right)
         {
-            return left.LanguageId == right.LanguageId;
```

Comment:
It's just convention that is hard coded into my brain. I got burned going `Equals` to `==` so many times due to not being careful with `==` usage that I stopped doing it.  Don't have a hard preference here though.